src/Exercism.Analyzers.CSharp/Analyzers/WeighingMachine/WeighingMachineComments.cs

```diff
@@ -0,0 +1,20 @@
+﻿
+using static Exercism.Analyzers.CSharp.Analyzers.Shared.SharedCommentParameters;
+
+namespace Exercism.Analyzers.CSharp.Analyzers.WeighingMachine
+{
+    internal class WeighingMachineComments
+    {
+        public static SolutionComment PropertyIsNotAutoProperty(string name) =>
+            new("csharp.weighingmachine.property_is_not_auto_property", new SolutionCommentParameter(Name, name));
+
+        public static SolutionComment PrecisionPropertyHasNonPrivateSetter(string name) =>
+            new("csharp.weighingmachine.property_setter_is_not_private", new SolutionCommentParameter(Name, name));
+
+        public static SolutionComment RoundMethodNotCalledInDisplayWeightProperty =
+            new("csharp.weighingmachine.round_called_in_display_weight");
+
+        public static SolutionComment PropertyBetterUseInitializer(string name) =>
+            new("csharp.weighingmachine.property_better_use_initializer", new SolutionCommentParameter(Name, name));
```

Comment:
I think this comment could actually also be useful for other exercises. As such, could you make this be a general comment? This means moving it to the `SharedComments.cs` class and having it start with `csharp.general`.